src/schemata/definitions/step/element.yml

```diff
@@ -74,15 +64,26 @@ properties:
       - type: number
         minimum: 1
         maximum: 12
-  dataScope:
-    type: object
-    $ref: '#/definitions/member-name' # measurement field
-    description: Scope of a measurement
-  inputChannels: # channels
-    type: array
-    items:
-      $ref: '#/definitions/step/element-objects/input-channel'
-    description: List of channels
+    examples:
+      - 6
+      - xs: 12
+        md: 6
+  oneOf:
```

Comment:
this is the cause of `Unknown keyword 0`.. you can not have a `oneOf` as the child of `properties`..

Reply:
So, IIUC, we need to give this oneOf a proper property name before using it. This means that we will need to change our templates being impacted by that.